english/pinch/go-out/follow-rabbit/follow-rabbit.md

```diff
@@ -1,3 +1,7 @@
 You jumped into the hole and slide down for quite some time, until you hit something soft.
 
-In the dim light, you try to stand and see where you are.
\ No newline at end of file
+In the dim light, you try to stand and see where you are.
+
+When your eyes adjust to the ambient light, you realize you're back in the 
+
+[marshmallow room](english/marshmallow.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.